lm_eval/tasks/jnli.py

```diff
@@ -0,0 +1,104 @@
+"""
+JGLUE: Japanese General Language Understanding Evaluation
+https://aclanthology.org/2022.lrec-1.317/
+
+JGLUE, Japanese General Language Understanding Evaluation, is built to measure the general NLU ability in Japanese. 
+JGLUE has been constructed from scratch without translation. 
+
+Homepage: https://github.com/yahoojapan/JGLUE
+"""
+from lm_eval.base import MultipleChoiceTask, rf
+
+_CITATION = """
+@inproceedings{kurihara-etal-2022-jglue,
+    title = "{JGLUE}: {J}apanese General Language Understanding Evaluation",
+    author = "Kurihara, Kentaro  and
+      Kawahara, Daisuke  and
+      Shibata, Tomohide",
+    booktitle = "Proceedings of the Thirteenth Language Resources and Evaluation Conference",
+    month = jun,
+    year = "2022",
+    address = "Marseille, France",
+    publisher = "European Language Resources Association",
+    url = "https://aclanthology.org/2022.lrec-1.317",
+    pages = "2957--2966",
+    abstract = "To develop high-performance natural language understanding (NLU) models, it is necessary to have a benchmark to evaluate and analyze NLU ability from various perspectives. While the English NLU benchmark, GLUE, has been the forerunner, benchmarks are now being released for languages other than English, such as CLUE for Chinese and FLUE for French; but there is no such benchmark for Japanese. We build a Japanese NLU benchmark, JGLUE, from scratch without translation to measure the general NLU ability in Japanese. We hope that JGLUE will facilitate NLU research in Japanese.",
+}
+"""
+
+
+
+class JNLIWithFintanPrompt(MultipleChoiceTask):
+    """
+    prompt template is taken from [ChatGPT vs BERT: どちらが日本語をより理解できるのか?](https://fintan.jp/page/9126/)
+    """
+    VERSION = 1.1
+    PROMPT_VERSION = 0.2
+    DATASET_PATH = "shunk031/JGLUE"
+    DATASET_NAME = "JNLI"
+    DESCRIPTION = "前提と仮説の関係をentailment、contradiction、neutralの中から回答してください。\n\n" + \
+        "制約:\n" + \
+        "- 前提から仮説が、論理的知識や常識的知識を用いて導出可能である場合はentailmentと出力\n" + \
+        "- 前提と仮説が両立しえない場合はcontradictionと出力\n" + \
```

Comment:
@mkshing generally LGTMeng 👍 

I made additional modification to the task description and resolved the conflict. Going to merge.